auth-web/src/services/keycloak.services.ts

```diff
@@ -24,6 +24,7 @@ export default {
   initSession () {
     configHelper.addToSession(SessionStorageKeys.KeyCloakToken, this.kc.token)
     configHelper.addToSession(SessionStorageKeys.KeyCloakRefreshToken, this.kc.refreshToken)
+    configHelper.addToSession(SessionStorageKeys.UserFullName, this.getUserInfo().firstName)
```

Comment:
Can we use the 'name' attribute which is basically the display name for users. First Name alone won't be sufficient I guess.

Reply:
Right, done.